apply-alacritty.py

```diff
@@ -1,23 +1,77 @@
 import io
 import json
 import sys
-from appdirs import user_config_dir
-from ruamel.yaml import YAML # use ruamel.yaml to preserve comments in config
+import os
+from ruamel.yaml import YAML  # use ruamel.yaml to preserve comments in config
 
 
+def get_conf_path():
+    # Determine system
+    # When we are in some Java world do extra checks
+    if sys.platform.startswith('java'):
+        import platform
+        os_name = platform.java_ver()[3][0]
+        if os_name.startswith('Windows'):  # "Windows XP", "Windows 7", etc.
+            system = 'win32'
+        else:  # anything that isn't windows ("darwin", "Linux", "SunOS", "FreeBSD", "Arch", etc.)
+            system = 'linux2'
+    else:
+        system = sys.platform
+
+    if system == 'win32':
+        # In windows alacritty config can only exist in one place
+        alacritty_path = os.path.expandvars(r'%APPDATA%\alacritty\alacritty.yml')
+        if os.path.exists(alacritty_path):
+            return alacritty_path
+    else:
+        # If it is not win32 it can exists in only a few other places
```

Comment:
https://github.com/alacritty/alacritty#configuration i could have gone with the same technique as the windows stuff here and done an `expandvars` thing, but if XDG_CONFIG_HOME is undefined it would try to look for the file in the root dir (`/alacritty/alacritty.yml` for example) which will probably lead to unwanted side-effects, permission prompts and the likes)